tests/unit/test_row.py

```diff
@@ -359,6 +359,28 @@ def test_commit(self):
         row.commit()
         self.assertEqual(table.mutated_rows, [row])
 
+    @mock.patch("google.cloud.bigtable.table.Table.mutate_rows")
```

Comment:
This patch is not used (you stub in a `_Table` instance).  Please remove it.

Reply:
Done.